dnf-behave-tests/fixtures/specs/dnf-ci-changelog/pkg-with-changelogs-1.0-1.spec

```diff
@@ -0,0 +1,20 @@
+Name:           pkg-with-changelogs
+Version:        1.0
+Release:        2
+
+License:        Public Domain
+URL:            None
+
+Summary:        A testing package with changelogs.
+
+%description
+A testing package.
+
+%files
+
+%changelog
+* Mon May 01 2023 DNF5 Team <pkgs@dnf.team> - 1.0-2
+- New downstream release 1.0-2
+
+* Fri Jan 20 2023 DNF5 Team <pkgs@dnf.team> - 1.0-1
+- New downstream release 1.0-1
```

Comment:
Can you either add an older changelog entry, or test with a newer date, so that the output differs from outputting the whole changelog?

Reply:
Of course, that's a good idea.